.travis.yml

```diff
@@ -7,3 +8,18 @@ before_script: bundle update
 script: "script/cibuild"
 sudo: false
 cache: bundler
+
+matrix:
+  exclude:
+    - rvm: 1.9.3
+      env: JEKYLL_VERSION=3.0.0.beta8
+    - env: JEKYLL_VERSION=2.4
+      rvm: 2.1
+    - rvm: 2.2
+      env: JEKYLL_VERSION=2.4
```

Comment:
@pathawks @parkr I pinned 2.4 to a single matrix because it sped up tests on a legacy version of the software we no longer support but needed to test because of Github Pages (I should have  pinned it to their version.) As for preventing it from building on 1.9.3 that's because we stripped all support for anything below 2.0 in 3.0 so I needed it to skip that matrix too.